README.md

```diff
@@ -44,36 +53,47 @@ configure your Purple Robot HTTP Upload Endpoint to `http://your-ip:9292/`
 ### Install Rails
 
     $ gem install rails
+    ...
 
 ### Create and configure the application
 
     $ rails new pr_endpoint
-    $ echo "gem 'purple_robot_data_capture_ruby', git: 'https://github.com/cbitstech/purple_robot_data_capture_ruby.git'" >> Gemfile
+    ...
+    $ echo "gem 'purple_robot_data_capture_ruby', \
+      git: 'https://github.com/cbitstech/purple_robot_data_capture_ruby.git'" \
```

Comment:
Yeah, I just didn't want to have to update the README every time the version changes.